pkgs/by-name/ca/candy-icons/package.nix

```
@@ -0,0 +1,42 @@
+{ lib
+, stdenvNoCC
+, fetchFromGitHub
+, gtk3
+}:
+
+stdenvNoCC.mkDerivation {
+  pname = "candy-icons";
+  version = "unstable-2023-10-03";
+
+  src = fetchFromGitHub {
+    owner = "EliverLara";
+    repo = "candy-icons";
+    rev = "bcf24f3308cc5e39f3e7d1bb43cb26f627f9ba8f";
```

```suggestion
    rev = "e4464d7b4d8e1821025447b2064b6a8f5c4c8c89";
```
Changed to the latest commit.